bucket/gcc.json

```diff
@@ -6,42 +6,42 @@
-                "http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-xxhash-0.7.3-1-any.pkg.tar.xz",
-                "http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-gdb-9.1-3-any.pkg.tar.xz"
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-binutils-2.34-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-crt-git-8.0.0.5685.ef310786-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-expat-2.2.9-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-gcc-9.3.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-gcc-fortran-9.3.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-gcc-libgfortran-9.3.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-gcc-libs-9.3.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-gettext-0.19.8.1-8-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-gmp-6.2.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-headers-git-8.0.0.5685.ef310786-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-isl-0.22.1-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libiconv-1.16-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libwinpthread-git-8.0.0.5680.0df6b89f-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-make-4.3-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-mpc-1.1.0-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-mpfr-4.0.2-2-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-pkg-config-0.29.2-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-windows-default-manifest-6.4-3-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-winpthreads-git-8.0.0.5680.0df6b89f-1-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-zlib-1.2.11-7-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libsystre-1.0.1-4-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libtre-git-r128.6fb7206-2-any.pkg.tar.xz",
+                "http://mirrors.ustc.edu.cn/msys2/mingw/x86_64/mingw-w64-x86_64-libtasn1-4.9-1-any.pkg.tar.xz",
```

Comment:
This file is not available any more, it is now `mingw-w64-x86_64-libtasn1-4.15.0-1-any.pkg.tar.xz`